src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ObjectWriter.cs

```diff
@@ -767,9 +784,10 @@ public static void EmitObject(string objectFilePath, IEnumerable<DependencyNode>
                 }
                 else
                 {
-                    objectWriter.SetSection(MethodCodeNode.UnixContentSection);
+                    managedCodeSection = MethodCodeNode.UnixContentSection;
                     objectWriter.SetSection(LsdaSection);
                 }
+                objectWriter.SetCodeSectionAttribute(managedCodeSection);
```

Comment:
The sentinels do not contain instructions. It makes sense to have standard sections also have the correct attributes instead of just for custom sections so things are more consistent. I filed issue https://github.com/dotnet/corert/issues/2913 for this.